src/platform/nrfconnect/OTAImageProcessorImpl.cpp

```diff
@@ -129,8 +155,15 @@ CHIP_ERROR OTAImageProcessorImpl::ProcessHeader(ByteSpan & block)
 
     if (mContentHeaderParser.IsInitialized() && !block.empty())
     {
-        OTAImageContentHeader header = {};
-        CHIP_ERROR error             = mContentHeaderParser.AccumulateAndDecode(block, header);
+        CHIP_ERROR error = mContentHeaderParser.AccumulateAndDecode(block, mContentHeader);
+        ChipLogDetail(SoftwareUpdate, "Found following DFU Images:");
+        for (uint8_t i = 0; i < mContentHeader.kMaxFiles; i++)
+        {
+            ChipLogDetail(SoftwareUpdate, "[%d]: \n \
```

Comment:
Please try not to use multi-line logs, they are more difficult to parse in testing. Also, you can use `%" PRIu32`, instead of "%d" as there are compiler warnings that validate if format string matches the provided data types.